src/System.Reflection.Metadata/src/System/Reflection/Metadata/Ecma335/Encoding/BlobEncoders.cs

```diff
@@ -74,24 +75,21 @@ public MethodSignatureEncoder PropertySignature(bool isInstanceProperty = false)
             return new MethodSignatureEncoder(Builder, hasVarArgs: false);
         }
 
-        public void CustomAttributeSignature(out FixedArgumentsEncoder fixedArguments, out CustomAttributeNamedArgumentsEncoder namedArguments)
+        [return: TupleElementNames(new[] { "FixedArguments", "NamedArguments" })]
```

Comment:
For the limited use case @tmat is using here, I think it's very solid. I'm not concerned about changing this, since I assume we'll eventually want S.R.M to be compiled with a 2.0.0 compiler, so we'll handle the encoding in the compiler.